docs/users/security/overview.md

```diff
@@ -0,0 +1,34 @@
+<!--
+order: 1
+-->
+
+# Overview
+
+Learn about security on Evmos {synopsis}
+
+The [Security Policy on Evmos](https://github.com/evmos/evmos/security/policy)
+includes a vulnerability disclosure policy and guidelines for
+reporting vulnerabilities in the Evmos platform.
```

Comment:
nit: "in the platform" sounds weird to me, I'd prefer either "on the platform" or "in the ecosystem"
```suggestion
reporting vulnerabilities in the Evmos ecosystem.
```